docs/sites.yml

```diff
@@ -10965,3 +10965,15 @@
     - E-commerce
   built_by: Annie Taylor Chen
   built_by_url: https://www.annietaylorchen.com/
+  - title: Inbound Latino
+  url: https://inboundlatino.com/
+  main_url: https://inboundlatino.com/
+  source_url: https://github.com/AnnieTaylorCHEN/PetiteandMinimalStore
```

Comment:
Is this the correct source URL? It appears to be the same as the entry before.

Reply:
Hello, you are right, it was the wrong URL but I fixed it.

Sorry.